### ISSUE TITLE
Searching for TCA shows barely related content

### ISSUE THREAD

**Issue by andreaskienast** [TYPO3-Documentation/t3docs-search-indexer#26]
If I search for "TCA" in the docs search, I get every extension having something with TCA listed. This is fine, but I'd expect to get the Core's TCA documentation at first.

**Comment by tmotyl** [comment]
I think this can be solved in many ways. 
1. improve TCA page content so its ranked higher
2. Use some mechanism to improve some popular phrases
- either by synonims
- Or by so other mechanism to boost certain pages on some phrase (eg by adding a high ranked „keywords” field) or even doing a redirect when you search for sth

Is there any other posibility @gr0 ?

**Comment by gr0** [comment]
Potential improvement for this issue should come from #18 - after introducing that change the newest version of the official core docs should appear higher. With appropriate boost they should be on top of the results for queries. That way when searching for `tca` the documents from the core documentation should high in the results and I think this is what we are aiming for.

In addition to that, a static boost could be included in all the queries to always boost documents that include `reference-coreapi` in the URL, something that could be done when working on #15. 

Of course improvements to the documents and their structure is always a plus and if there are possibilities that should happen as well. 

I would avoid doing redirects for now - this is clearly a relevancy problem and with redirects we are only hiding the issue. I assume `tca` is just an example of a problematic query and we can fine a dozen of cases will result in the core documentation not being on top in the results pages.

**Comment by tmotyl** [comment]
deployed #18, the results seems beter now, can you confirm @andreasfernandez ?

**Comment by andreasfernandez** [comment]
It still ranks [ckeditor TCA](https://docs.typo3.org/c/typo3/cms-rte-ckeditor/master/en-us/Configuration/ConfigureTypo3.html#tca) higher than [TCA reference](https://docs.typo3.org/m/typo3/reference-tca/master/en-us/Index.html#tca-reference), but it's a lot better now :+1:

**Comment by tmotyl** [comment]
yep, both are official core manuals, so they have the same boosting see https://github.com/TYPO3-Documentation/t3docs-search-indexer/commit/597925c477c9bab2d71cec13e6c428e36060b88b#diff-7b8c1d52de730061d038585bff361c5093291d499369080abf9d5c97e14e4c52R203

**Comment by tmotyl** [comment]
@andreasfernandez can you test/check now ckeditor is first result, but following results are very good.
To kick ckeditor out of the first place, the chapter title should be changed -> see https://github.com/TYPO3/typo3/pull/352

**Comment by tmotyl** [comment]
lets remove v10 from boosting. 
https://github.com/TYPO3-Documentation/t3docs-search-indexer/blob/main/src/Repository/ElasticRepository.php#L206C98-L206C98

**Comment by linawolf** [comment]
Results look good now
